src/Storefront/Framework/SystemCheck/SaleChannelsReadinessCheck.php

```diff
@@ -136,7 +136,12 @@ private function generateDomainUrl(string $url): string
 
     private function whileTrustingAllHosts(callable $callback): Result
     {
-        $trustedHosts = Request::getTrustedHosts();
+        // Remove '{' from start and '}i' from end, applied by Request::setTrustedHosts.
```

Comment:
do we need to do that, as it would be applied a second time?

Reply:
Yes, 100%